debian/mariadb-server.postrm

```diff
@@ -14,13 +14,22 @@ ${DEBIAN_SCRIPT_TRACE:+ echo "#42#DEBUG# RUNNING $0 $*" 1>&2 }
 
 MYADMIN="/usr/bin/mysqladmin --defaults-file=/etc/mysql/debian.cnf"
 
+MARIADBD_USERS="root"
```

Comment:
removing file like - https://salsa.debian.org/mariadb-team/mariadb-server/-/merge_requests/75 ?

Reply:
There is now PR #3210 which imports needed changes from Debian Salsa and reworks them to MariaDB upstream repo. It should get in first and then this should be adapted to changes.